.github/workflows/build-and-run-model.yaml

```diff
@@ -27,11 +27,14 @@ jobs:
       # required in order to allow the reusable called workflow to push to
       # GitHub Container Registry
       packages: write
-    uses: ccao-data/actions/.github/workflows/build-and-run-batch-job.yaml@main
+    uses: ccao-data/actions/.github/workflows/build-and-run-batch-job.yaml@jeancochrane/support-container-env-vars-in-batch-jobs
```

Comment:
This is just for the purposes of testing; I'll plan to switch this back to `main` before merging.